go.mod

```diff
@@ -0,0 +1,47 @@
+module github.com/CMSgov/bcda-app
+
+go 1.15
+
+require (
```

Comment:
Wanting to confirm, we've double checked all of the direct dependencies here and all of the versions match up with what was existing in the Gopkg.lock file?

Reply:
The ones called out in the feedback section above are the ones I'm not sure of - `go-agent` seems to use an import path both with and without `/v3` (see usage in `metrics_test.go`; all other instances us the `/v3` version); `testify` and `crypto` I only caught this morning that there are different versions between the two files so double-checking those

Reply:
Let me check those packages you listed and I'll respond back here. Thanks for listing those out!

Reply:
From offline conversation, confirmed correct usage of `go-agent` and `vegeta`; `crypto` and `testify` have been updated to match the revision/version from the most recent `Gopkg.lock` file. Made a note to discuss future work around upgrading some of these dependencies after we've moved over to Go Modules.